int64 sender_robot_id
int64 assigned_robot_id
float64 x
float64 y
float64 z